src/opm/output/eclipse/EclipseIO.cpp

```diff
@@ -284,7 +284,7 @@ RestartValue EclipseIO::loadRestart(SummaryState& summary_state, const std::vect
     const int report_step                = initConfig.getRestartStep();
     const std::string filename           = ioConfig.getRestartFileName( initConfig.getRestartRootName(),
                                                                         report_step,
-                                                                        false );
+                                                                        true );
```

Comment:
This looks strange to me.  The `bool` flag of [`getRestartFileName()`](https://github.com/OPM/opm-common/blob/0756cd93ada466a7b96dfbe34d33103ab94981af/src/opm/parser/eclipse/EclipseState/IOConfig/IOConfig.cpp#L206-L208) specifies whether to create the filename for an output  file (`true`) or an input file (`false`).  Since `loadRestart()` is about reading from an existing result file, I think this should still be specified as `false`.

Incidentally, this is my favourite example for why we should never use naked `bool`s as parameters in a client-facing APIs.

Reply:
If I did not change this It tried to read from DATFILE.X000N. It probably should be figured out from how the outputfil was written.

Reply:
> If I did not change this It tried to read from `DATFILE.X000N`.

Ah, that **probably** means that the `.DATA` file you used for input does not have
```
UNIFIN
```
to specify that the restart files are unified.

> It probably should be figured out from how the outputfile was written.

Possibly, yes.  It's fairly easy to do so, especially since we have C++17 and can rely on filesystem support in the standard library.